index.js

```diff
@@ -62,72 +29,56 @@ function prepareLogs(eventData, log_raw_event) {
     , meta: {
         owner: eventData.owner
       , filters: eventData.subscriptionFilters
-      }, line: JSON.stringify({message: event.message, ...eventMetadata})
+      }
+    , line: event.message
     }
 
-    if (log_raw_event) {
-      eventLog.line = event.message
-      eventLog.meta = {...eventLog.meta, ...eventMetadata}
-    }
+    eventLog.meta = {...eventLog.meta, ...eventMetadata}
 
     return eventLog
   })
 }
 
 // Ship the Logs
-function sendLine(payload, config, callback) {
-  // Check for Ingestion Key
-  if (!config.key) return callback('Missing Pipeline Ingestion Key')
+async function sendLine(payload, callback) {
 
   // Prepare HTTP Request Options
   const options = {
-    method: 'POST'
-  , body: JSON.stringify({
-      e: 'ls'
-    , ls: payload
-    })
+    body: JSON.stringify(payload)
   , headers: {
-      'user-agent': config.UserAgent
-    , 'Authorization': config.key
+      'user-agent': config.get('user-agent')
+    , 'Authorization': config.get('pipeline-key')
     }
   , timeout: {
-      request: MAX_REQUEST_TIMEOUT_MS
+      request: config.get('max-request-timeout')
     }
   }
 
-  // Flush the Log
-  asyncRetry({
-    times: MAX_REQUEST_RETRIES
-  , interval: (retryCount) => {
-      return REQUEST_RETRY_INTERVAL_MS * Math.pow(2, retryCount)
-    }, errorFilter: (errCode) => {
-      return DEFAULT_HTTP_ERRORS.includes(errCode) || errCode === 'INTERNAL_SERVER_ERROR'
-    }
-  }, (reqCallback) => {
-    return got(PIPELINE_URL, options, (error, response, body) => {
-      if (error) {
-        return reqCallback(error.code)
+  try {
+    const resp = await got.post(
+      config.get('pipeline-url')
+    , options
+    , {
+        retry: {
+          limit: config.get('max-request-retries')
+        , maxRetryAfter: config.get('request-retry-interval'), methods: ['GET', 'POST']
```

Comment:
NIt: `methods` should be on the next line.

Do we need the method declaration here when post is explicitly called? Is `GET` a necessary definition?